Dockerfile-hydroshare.template

```diff
@@ -0,0 +1,10 @@
+
+RUN usermod -u 1000 hydro-service
+RUN groupmod -g 1000 storage-hydro
```

Comment:
hardcoding UID and GID to 1000 would be problematic since UID and GID may not be 1000 such as the hydro-service account in our deployment VMs. It'd be good to read the UID and GID from a config file since I saw there are multiple places where this UID and GID need to be updated if they are different from the default 1000.

Reply:
Sure, thanks for review, I did not check carefully this part

Reply:
Does this need to be corrected throughout hs_docker_base ?